test/react_test_utils_test.dart

```diff
@@ -59,27 +59,17 @@ void main() {
       expect(domNode.text, equals(''));
     });
 
-    void testEvent(
-        void event(dynamic instanceOrNode, Map eventData), String eventName) {
-      Map eventData;
-      int fakeTimeStamp;
-
-      setUp(() {
-        fakeTimeStamp = eventName.hashCode;
-        eventData = {
-          'type': eventName,
-          'timeStamp': fakeTimeStamp,
```

Comment:
Can we keep `eventData` and `timeStamp` in? I believe they're needed to test coverage of the eventData arg.